manifests/charts/istio-control/istio-discovery/files/gateway-injection-template.yaml

```diff
@@ -231,9 +231,17 @@ spec:
           audience: {{ .Values.global.sds.token.aud }}
   {{- if eq .Values.global.pilotCertProvider "istiod" }}
   - name: istiod-ca-cert
+  {{- if (.Values.pilot.env).ENABLE_CLUSTER_TRUST_BUNDLE_API }}
```

Comment:
nit: probably should check its true; this would pass if it was `false`?

Reply:
done